src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier;
+
+    public Lottery(ColorSupplier colorSupplier) {
+        this.colorSupplier = colorSupplier;
+    }
+
+    public Ball getRandomBall() {
+        String color = colorSupplier.getRandomColor().name();
+        int number = new Random().nextInt(100) + 1;
```

Comment:
add constant for 100 - it is magic number
make Random class level variable just like ColorSupplier